### ISSUE TITLE
Could not load session: 404: Not Found

### ISSUE THREAD

**Issue by joetcochran** [cschleiden/azure-boards-estimate#93]
Running version 2.2.3 against Azure DevOps Server Update 1.

When navigating into the session I just created, I get an error stating "Could not load session: 404: Not Found"

Additional info from Developer tools indicate some 404's on a call to "Documents"

I have moved my DefaultCollection from the "normal" place of "myserver.com/DefaultCollection" to "myserver.com/devops/DefaultCollection" - could this be why? Is there some config in the extension I need to change to account for this?

**Comment by PatrickBig** [comment]
I'm having the same issue

**Comment by jakekobes** [comment]
Yeah I have the same issue, we just did an in-place upgrade of TFS 2017 Update 3.1 to Azure DevOps Server 2019 Update 1 and installed this extension and throws the same error on creating a new session.

**Comment by thePilotPirX** [comment]
Same issue, both on Azure DevOps Server 2019 Update 1 and Azure DevOps Server 2019 Update 1.1

**Comment by diogopalexandre** [comment]
Same issue, Azure DevOps Server 2019 Update 1.1

**Comment by sjsurfer** [comment]
Same issue. Estimate extension 2.2.3.  Azure Devops Server Dev17.M153.3 (Azure Devops Server 2019 Update 1).  Interestingly, it is working for some of my projects, but not others.  I haven't identified a difference.

**Comment by TGW-byte** [comment]
Same issue, please advise how to fix it.

**Comment by assofohdz** [comment]
I'm in the online Azure DevOps and getting this when trying to join a session I created

**Comment by feynmanq** [comment]
Same issue.
17.153.29522.3 (AzureDevOps2019.Update1.1)
Version Dev17.M153.5

**Comment by jmesserer** [comment]
Same issue.

17.153.29207.5 (AzureDevOps2019.Update1)
Version Dev17.M153.3

Extension Version: 2.2.3 (installed 04/15/2020)

**Comment by parthibanj007** [comment]
Same issue.

**Comment by Samuel-e1** [comment]
Same issue.
On premise, Version Dev17.M153.5

**Comment by melvis137** [comment]
Same issue.
17.153.29522.3 AzureDevOps2019.Update.1.1
